cmd/crossplane/core/core.go

```diff
@@ -298,17 +301,46 @@ func (c *startCommand) Run(s *runtime.Scheme, log logging.Logger) error { //noli
 		// fleshed out, implement a registration pattern similar to scheme
 		// registrations.
 		if err := xrd.SetupWebhookWithManager(mgr, o); err != nil {
-			return errors.Wrap(err, "cannot setup webhook for compositeresourcedefinitions")
+			return errors.Wrap(err, "Cannot setup webhook for compositeresourcedefinitions")
```

Comment:
Is this intentional? Shouldn't error messages start with lower case?

Reply:
I saw other errors above starting with capital letters I thought that made sense give that these are root level errors so I just adapted, but I now noticed there are a few with lowercase, no strong opinion here I can switch them all to lowercase.

Reply:
No strong opinion either, but all lowercase would be my preference.

Reply:
Done 👌